RELEASE-NOTES.txt

```diff
@@ -1,3 +1,11 @@
+1.1.0
+=====
+* Introduced a new feature to export and import sites
+* Translated the tooltip for the close button on the modal #454
+* Added support for Ukrainian language #452
+* Increased space limit to 250 MB #446
```

Comment:
```suggestion
* Increased demo site space limit to 250 MB #446
```

Reply:
Updated in https://github.com/Automattic/studio/pull/472/commits/b881e0019a74b1d3c87f78462e1d900dafcd0ba8